openBCIBoard.js

```diff
@@ -238,24 +322,110 @@ function OpenBCIFactory() {
         });
     };
 
+    /**
+     * @description Should be used to send data to the board
+     * @param data
+     * @returns {Promise} if signal was able to be sent
+     * @author AJ Keller (@pushtheworldllc)
+     */
+    OpenBCIBoard.prototype._writeAndDrain = function(data) {
```

Comment:
does serialPort have a built-in writeAndDrain. if so, what makes this one different from that?